plugins/GigPlayer/CMakeLists.txt

```diff
@@ -3,15 +3,34 @@ if(LMMS_HAVE_GIG)
 	include_directories(SYSTEM ${GIG_INCLUDE_DIRS})
 	SET(CMAKE_AUTOUIC ON)
 
+	# Version checking logic disabled for msvc as vcpkg guarantees gig > 4.3
```

Comment:
I think it would be better to do version checking unconditionally as a sanity check. It's less complicated that way, makes fewer assumptions about our CI, and would catch any future mistakes.

Plus, if someone tried to build lmms on their own with MSVC, we'd want it to detect if they had an unsupported libgig version.

Reply:
The version checking is broken on MSVC and causes a build fail, that's why I brought in this workaround.